docs/source/system_reference_guide/jobs_maappy.ipynb

```diff
@@ -135,6 +135,61 @@
+    "\n",
+    "```\n",
+    "r = maap.listJobs(\"mlucas\")\n",
+    "print(r.text)\n",
+    "```\n",
+    "\n",
+    "Here is a sample truncated output:\n",
+    "\n",
+    "```\n",
+    "{\"code\": 200, \"jobs\": [{\"b308b3cd-8848-4154-b682-aaf3d39734ee\": {\"resource\": \"job\", \"payload_hash\": \"0f217c35a6bdacf46b164106af9ab473\", \"job_id\": \"job-dps_tutorial_mlucas_test__main-20240626T172521.763123Z\", \"status\": \"job-completed\", \"context\": {\"tag\": \"test-new\", \"_prov\": {\"wasGeneratedBy\": \"task_id:a4761c87-dc68-4511-9a3e-e628446f1e44\", \"wasDerivedFrom\": [\"url:https://photojournal.jpl.nasa.gov/tiff/PIA00127.tif\"]}, \"container_image_name\": \"container-dps_tutorial:main\", \"outsize\": 30, \"container_specification\": {\"version\": \"main\", \"digest\": \"sha256:e539186c9e76fb923da7b3bfb6a3c700403e5302a71c3e8ba8635b7f216ab7e3\", \"id\": \"container-dps_tutorial:main\", \"url\": \"s3://s3-us-west-2.amazonaws.com/maap-dit-registry/container-dps_tutorial:main.tar.gz\"}, \"container_mappings\": {\"$HOME/.aws\": \"/home/ops/.aws\", \"$HOME/.netrc\": \"/home/ops/.netrc\", \"/tmp\": [\"/tmp\", \"rw\"]}, \"_command\": \"/app/dps_wrapper.sh '/app/dps_tutorial/gdal_wrapper/run_gdal.sh' 'output.tif' '30'\", \"localize_urls\": [{\"url\": \"https://photojournal.jpl.nasa.gov/tiff/PIA00127.tif\"}], \"output_file\": \"output.tif\", \"_disk_usage\": \"1GB\", \"container_image_url\": \"s3://s3-us-west-2.amazonaws.com/maap-dit-registry/container-dps_tutorial:main.tar.gz\", \"container_image_id\": \"sha256:e539186c9e76fb923da7b3bfb6a3c700403e5302a71c3e8ba8635b7f216ab7e3\", \"job_specification\": {\"post\": [\"hysds.triage.triage\"], \"resource\": \"jobspec\", \"container\": \"container-dps_tutorial:main\", \"soft_time_limit\": 86400, \"time_limit\": 86400, \"recommended-queues\": [\"maap-dps-worker-8gb\"], \"job-version\": \"main\", \"command\": \"/app/dps_wrapper.sh '/app/dps_tutorial/gdal_wrapper/run_gdal.sh'\", \"params\": [{\"destination\": \"localize\", \"name\": \"input_file\", \"value\": \"https://photojournal.jpl.nasa.gov/tiff/PIA00127.tif\"}, {\"destination\": \"positional\", \"name\": \"output_file\", \"value\": \"output.tif\"}...\n",
+    "```\n",
+    "\n",
+    "Users may provide optional query parameters to filter the list on desired fields. The following example will return a list of jobs submitted by user `mlucas` \n",
+    "that all ran the `job-dps_tutorial_mlucas:main` algorithm:\n",
+    "\n",
+    "```\n",
+    "r = maap.listJobs(\"mlucas\", algo_id=\"job-dps_tutorial_mlucas\", version=\"main\")\n",
+    "```\n",
+    "\n",
+    "Users may also request just the list of jobs id's and their corresponding tags rather than full job details. This is controlled using the `get_job_details` option. By default `get_job_details` is set to `True`, but if set to `False` will return a compact list, as shown in this example:\n",
```

Comment:
how did we decide that get_job_details=False only returns JobID and Tag? Why not, for example, status?

Reply:
We didn't really decide what gets returned in the abbreviated response. That's what the Mozart API returns. We could append arbitrary fields from the MAAP API, but we would have to request the full job details and parse the desired fields and add that to the MAAP API response.